locale/foreman_discovery.pot

```diff
@@ -77,19 +147,46 @@ msgstr ""
 msgid "Facts refreshed for %s"
 msgstr ""
 
-msgid "Failed to import facts for Host::Discovered"
+msgid "Failed to auto provision host %s: %s"
+msgstr ""
+
+msgid "Failed to import facts for discovered host"
 msgstr ""
 
-msgid "Failed to import facts for Host::Discovered: %s"
+msgid "Failed to import facts for discovered host: %s"
 msgstr ""
 
-msgid "Failed to reboot: %s"
+msgid "Failed to reboot host %s"
+msgstr ""
+
+msgid "Failed to reboot host %{hostname} with error %{error_message}"
 msgstr ""
 
 msgid "Failed to refresh facts for %s"
 msgstr ""
 
-msgid "Imported Host::Discovered"
+msgid "Host"
+msgstr ""
+
+msgid "Host %s was provisioned with rule %s"
```

Comment:
Shouldn't use two %s within one string - I suggest you fix the offending code and re-extract.